lib/components/artist/artworks/index.js

```diff
@@ -30,7 +30,7 @@ class Artworks extends React.Component {
       return this.renderSection('Works',
                                 other_count,
                                 this.props.artist.not_for_sale_artworks,
-                                () => { return ({ availability: 'IS_NOT_FOR_SALE'}) },
+                                { availability: 'IS_NOT_FOR_SALE' },
```

Comment:
This arg _was_ a query params function, which would return values that could then be used inside your actual query later ( in this case that lets it apply the availability scope ) 

Now it is the actual object - this object is used as a key to indicate when to reset the component. Docs are below in the source.